extensions/sql-database-projects/package.json

```diff
@@ -295,7 +300,11 @@
         },
         {
           "command": "sqlDatabaseProjects.publish",
-          "when": "false"
+          "when": "!config.sqlDatabaseProjects.enablePreviewFeatures"
+        },
+        {
+          "command": "sqlDatabaseProjects.publishDialog",
+          "when": "!azdataAvailable && config.sqlDatabaseProjects.enablePreviewFeatures"
```

Comment:
Shouldn't these also be false?  Entering this flow from the command palette doesn't make much sense since you need the context of which sqlproject

Reply:
Right! publish can only be done from project node, i shouldn't have these changes, thanks for noticing. removed the conditions.